.yamato/yamato-ci.yml

```diff
@@ -9,6 +9,9 @@ linux:
     - git submodule update
     - mono bee.exe tundra2::linux::master
     - mono bee.exe tundra2-unittest-report::linux::master
+    - mono bee.exe artifacts/for-stevedore/tundra-linux-x64.zip
+    - curl -sSo StevedoreUpload.exe "$STEVEDORE_UPLOAD_TOOL_URL"
+    - mono StevedoreUpload.exe --append-manifest=build/linux-x86_64/manifest.stevedore --version-len=12 --repo=public --version="$GIT_REVISION" artifacts/for-stevedore/*
```

Comment:
Is that manifest.stevedore file something that you're asking StevedoreUpload to generate, or is it something that has been built by some build step outside this PR?